HtmlCssSample.css

```diff
@@ -0,0 +1,188 @@
+body {
+    background-color: rgb(35, 56, 76);
+    display: flex;
+    margin: 0;
+}
+
+nav {
+    position: fixed;
+    background-color: white;
+    height: 10%;
+    width: 100%;
+    display: flex;
+}
+
+.logo {
+    display: flex;
+    justify-content: flex-start;
+    margin-left: 2%;
+    margin-top: 1%;
+    width: 10%;
+    height: 50%;
+}
+
+img {
+    height: 100%;
+}
+
+div {
+    display: inline-flex;
+    justify-content: space-evenly;
+    width: 60%;
+    height: 100%;
+    padding-left: 40%;
+}
+
+p {
+    margin-top: 2%;
+    height: 30%;
+}
+
+p:hover {
+    background-color: rgb(235, 233, 233);
+    cursor: pointer;
+}
+
+.buttons {
+    margin-top: 2%;
+}
+
+.btn-sign-in {
+    border-radius: 5px;
+    height: 35%;
+}
+
+.btn-register {
+    border-radius: 5px;
+    height: 35%;
+    background-color: black;
+    color: white;
+}
+
+.btn-sign-in:hover {
+    cursor: pointer;
+}
+
+.btn-register:hover {
+    cursor: pointer;
+}
+
+.html-heading {
+    font-size: 300%;
+    color: white;
+}
+
+.css-heading {
+    font-size: 200%;
+    color: white;
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 15%;
+    width: 100%;
+    height: 100%;
+}
+
+header {
+    margin-right: 3%;
+    margin-bottom: 2%;
+}
+
+.form-body {
+    background-color: rgb(245, 245, 245);
+    width: 20%;
+    height: 80%;
+    border-radius: 10px;
+    box-shadow: 10px 5px 5px rgb(131, 130, 130);
+    padding-top: 1%;
+}
+
+.input-details {
+    display: flex;
+    width: 90%;
+    margin-left: 3%;
+    margin-bottom: 10%;
+    border-radius: 5px;
+}
+
+input {
+    padding-top: 3%;
+    padding-bottom: 3%;
+}
+
+label {
+    margin-left: 15px;
+}
+
+.btn-submit-details {
+    width: 90%;
+    margin-left: 3%;
+    margin-bottom: 10%;
+    padding-left: 5%;
+    background-color: black;
+    color: rgb(255, 255, 255);
+    border-radius: 5px;
+    padding-top: 3%;
+    padding-bottom: 3%;
+}
+
+.btn-submit-details:hover {
+    background-color: rgb(236, 82, 82);
+    color: rgb(224, 210, 66);
+    cursor: pointer;
+}
+
+footer {
+    position: static;
+    background-color: rgb(255, 255, 255);
+    margin-top: 2%;
+    width: 100%;
+    padding-bottom: 20%;
+    display: inline-flex;
+    justify-content: space-between;
+}
+
+.footer-main-logo-section {
+    display: flex;
+    flex-direction: column;
+    margin-top: 5%;
+    padding-left: 2%;
+    width: 20%;
+}
+
+.link-logos-section {
+    display: flex;
+    flex-direction: row;
+    margin-top: 5%;
+    padding-left: 2%;
+    width: 20%;
+}
+
+.site-footer-logo {
+    height: 10%;
+    width: 10%;
+}
+
+.footer-link-logos {
+    height: 50%;
+    width: 50%;
+}
+
+table {
+    margin-top: 2%;
```

Comment:
Try to use the flex styles maximum instead of margin properties

Reply:
Resolved